lms/static/sass/views/_teams.scss

```diff
@@ -346,6 +346,106 @@
     }
   }
 
+  .team-profile {
+    .discussion-module {
+      width: 60%;
+      @include float(left);
+    }
+    .team-detail {
+      width: 30%;
+      @include float(left);
+      @include margin($baseline, 0, $baseline, $baseline + $baseline/2);
+
+      div, h4 {
```

Comment:
It would be great to avoid element styling here by using something like .team-detail-header perhaps?